sql/core/pom.xml

```diff
@@ -69,6 +69,11 @@
       <version>2.3.0</version>
     </dependency>
     <dependency>
+      <groupId>org.jodd</groupId>
+      <artifactId>jodd-core</artifactId>
+      <version>${jodd.version}</version>
+    </dependency>
```

Comment:
FYI most systems don't support leap seconds. I'm not sure why we'd want to support them here...